scene/resources/texture.cpp

```diff
@@ -277,7 +277,7 @@ bool ImageTexture::is_pixel_opaque(int p_x, int p_y) const {
 				img = decom;
 			}
 			alpha_cache.instantiate();
-			alpha_cache->create_from_image_alpha(img);
+			alpha_cache->create_from_image_alpha(img, p_threshold);
```

Comment:
This is not great, it means whoever calls the function first will generate it (and cache it) with a different mask. I also think the current threshold should be good enough for most cases (not entirely transparent basically), so I am not sure why it should be changed)

Reply:
It needs to be changed for the issue I linked. For that reason it had to be configurable first, hence all the changes. Is there a better way to fix the issue?